assembler.ts

```diff
@@ -359,7 +359,11 @@ class Assembler {
                 `Unknown pseudo instruction ${instr.op} at line ${instr.lineno}`
               );
           }
-          code.splice(i, 1);
+          if (typeof instr.next == "string") {
```

Comment:
Shouldn't this be `if (instr.next != null)`? E.g. what if there's a `.ret` right after `.out`

Reply:
Done